docs/policies.md

```diff
@@ -97,7 +97,13 @@ Catalyst provides a healthcare reimbursement of **$150 a month** to members and
 
 **Provided to:** Employee-Members only
 
-Catalyst employees can contribute to a tax-deferred [Vanguard SIMPLE IRA](https://investor.vanguard.com/small-business-retirement-plans/simple-ira) and the co-op will match 3% of the employee’s wages. The IRS limit on annual employee contributions as of 2022 is $14,000. Here is a good summary of [how a SIMPLE IRA works](https://www.paychex.com/articles/employee-benefits/what-is-a-simple-ira) . Employees can contribute to the SIMPLE IRA and still contribute to a traditional individual IRA or Roth IRA.
+Catalyst employees can contribute to a [Vangaurd Solo 401(k) (also known as an Individual 401k or i401(k))](https://investor.vanguard.com/accounts-plans/small-business-retirement-plans/individual-solo-401k) and the co-op will match 10% of the employee’s wages. Catalyst members are eligible for this type of retirement account because all employees are also owners.
+
+Employees can defer 100% of their compensation up to $22,500 for 2023 ($30,000 for employees age 50 or older). Employers can contribute up to 25% of compensation not to exceed $66,000 for the 2023 tax year. **Employees can make pre-tax or Roth (after-tax) contributions to the 401(k)**. However, all **employer** contributions are made pre-tax.
+
+Employees can contribute to the Solo 401(k) and still contribute to a traditional individual IRA or Roth IRA.
```

Comment:
I'm assuming this is still true. Do we think it's relevant given 401(k)s and IRAs are different types of retirement accounts?

Reply:
I feel like it's probably a given that they can put money in a traditional IRA account too. I think it's only worth mentioning if you can actually contribute to them via your withheld wages (i.e., Catalyst puts them there vs the 401k).

Reply:
[Looks like solo 401ks do not impact your ability to make Roth IRA contributions](https://www.mysolo401k.net/solo-401k/contribute-to-a-solo-401k-roth-ira-and-traditional-ira-for-2021/#:~:text=As%20long%20as%20your%20modified,making%20Roth%20Solo%20401k%20contributions.):
> The ability to contribute to your Roth solo 401k is not impacted if you also make Roth IRA contributions nor if your self-employment income is over a certain limit. As long as your modified AGI is not over a certain limit, you can also make Roth IRA contributions in addition to making Roth Solo 401k contributions. In short, you may not be able to also contribute to a Roth IRA if you make too much earned income.  What is more the conversion of voluntary after-tax solo 401k funds to the Roth IRA does not impact your annual Roth IRA contribution limit.